apps/condo/pages/phone/[number]/index.tsx

```diff
@@ -368,34 +438,59 @@ const ClientCardTabContent = ({
     contact = null,
     showOrganizationMessage = false,
     organization,
+    sectionName,
+    type,
+    sectionType,
 }) => {
     const intl = useIntl()
     const ShowAllPropertyTicketsMessage = intl.formatMessage({ id: 'pages.clientCard.showAllPropertyTickets' })
-    const ContactTicketsMessage = intl.formatMessage({ id: 'pages.clientCard.contactTickets' })
     const CreateTicketMessage = intl.formatMessage({ id: 'CreateTicket' })
     const EditContactMessage = intl.formatMessage({ id: 'pages.clientCard.editContact' })
+    const LastTicketsLoadedMessage = intl.formatMessage({ id: 'pages.clientCard.lastTicketsLoaded' }, { count: MAX_TABLE_SIZE })
+    const OpenAllTicketsMessage = intl.formatMessage({ id: 'pages.clientCard.openAllTickets' })
+
+    const tabsItems: tabsItems = useMemo(()=>tableTabs.map(el => ({
+        label: intl.formatMessage({ id: `pages.clientCard.table.tabs.${el}` }),
+        key: el,
+    })), [intl])
+
+    const [currentTableTab, setCurrentTableTab] = useState<TableTabs>('address')
 
     const router = useRouter()
 
     const { offset } = parseQuery(router.query)
     const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
-    const {
-        loading: isTicketsFetching,
-        count: total,
-        objs: tickets,
-    } = Ticket.useObjects({
-        sortBy: TICKET_SORT_BY,
-        where: searchTicketsQuery,
-        first: DEFAULT_PAGE_SIZE,
-        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
-    }, {
+
+    const searchQuery = useMemo(() => {
+        if (currentTableTab === 'entrance') {
+            return { ...searchTicketsQuery, contact: undefined, unitName: undefined, unitType: undefined,  sectionName, sectionType }
+        }
+
+        if (currentTableTab === 'home') {
+            return { ...searchTicketsQuery, contact: undefined, unitName: undefined, unitType: undefined }
+        }
```

Comment:
I think it's better to create a new object instead of overwriting `searchTicketsQuery` with undefined. It will be more obvious